### ISSUE TITLE
[A11y] MODX Manager Accessibility

### ISSUE THREAD

**Issue by jenswittmann** [modxcms/revolution#16612]
### Summary
The MODX Manager is based on ExtJS and is not accessible for keyboard navigation and screen readers. The manager should therefore also be accessible to the employees of a company who maintain the website.

See this for public discussion: https://community.modx.com/t/modx-manager-accessibilty/7990

### Why is it needed?
The [Barrierefreiheitsstärkungsgesetz (BFSG) 2](https://www.aktion-mensch.de/inklusion/barrierefreiheit/barrierefreie-website) comes into force on 28 June 2025.

### Suggested solution(s)
Bring Keyboardnavigation and Screenreadertest in the MODX Dev Workflow.
 
### Related issue(s)/PR(s)
Commits/PR will be prefixed with `[A11y]`.

**Comment by rthrash** [comment]
@jenswittmann looks like this may have been addressed. Are there still outstanding items that should be handled still?

It looks like you closed #16614, perhaps those items?

**Comment by jenswittmann** [comment]
@rthrash here are lots of other things to do in the future. Like screenreader annoucement, aria states, color contrast and so on. Maybe keep this as a parent issue to track the sub-issues (the new GitHub Feature)?

Here is a small issue by @Jako that needs to be resolved in the next release and could be tracked as a sub-issue with a separate PR? https://github.com/modxcms/revolution/pull/16613#issuecomment-2717788808

There was also a conversation between myself and Henk Everts before the Snowup to get rid of the ExtJS side by side. So new features will be developed on VanillaJS and have a better accessibility than trying to change the old ExtJS. But this needs more discussion how to do it 🤔

**Comment by Jako** [comment]
There are still some z-index issues left in 3.1.2. I.e. a switch is shown in front of the menu:

<img width="672" alt="Image" src="https://github.com/user-attachments/assets/a79b7f73-b968-4b13-8d96-9a74e047b4c7" />
